### ISSUE TITLE
Package.getActivationCommands is deprecated.

### ISSUE THREAD

**Issue by pavlo-yuriychuk** [smashwilson/merge-conflicts#155]
Use `activationCommands` instead of `activationEvents` in your package.json
Commands should be grouped by selector as follows:
```json
  "activationCommands": {
    "atom-workspace": ["foo:bar", "foo:baz"],
    "atom-text-editor": ["foo:quux"]
  }
```
```
Package.getActivationCommands (/usr/share/atom/resources/app.asar/src/package.js:808:9)
Package.hasActivationCommands (/usr/share/atom/resources/app.asar/src/package.js:733:20)
<unknown> (/usr/share/atom/resources/app.asar/src/package.js:187:24)
Package.measure (/usr/share/atom/resources/app.asar/src/package.js:165:15)
Package.load (/usr/share/atom/resources/app.asar/src/package.js:179:12)
PackageManager.loadPackage (/usr/share/atom/resources/app.asar/src/package-manager.js:372:14)
```

Atom version 1.0

**Comment by smashwilson** [comment]
This was also [corrected](https://github.com/smashwilson/merge-conflicts/pull/132/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2) by #132 and released in 1.3.0.

Thank you for the reports, though, it's good to double-check these things :grin:

**Comment by smashwilson** [comment]
@pavlo-yuriychuk: I'm going to close these issues because I've verified that all of the deprecations have been addressed in more recent package versions. If I'm wrong and you're still seeing any of them, even on 1.3.2, please re-open and we can investigate further to figure out why. :bow:
